packages/core/admin/admin/src/translations/en.json

```diff
@@ -651,6 +651,7 @@
   "content-manager.components.FiltersPickWrapper.PluginHeader.description": "Set the conditions to apply to filter the entries",
   "content-manager.components.FiltersPickWrapper.PluginHeader.title.filter": "Filters",
   "content-manager.components.FiltersPickWrapper.hide": "Hide",
+  "content-manager.components.Filters.usersSelect.label": "Search and select an user to filter by",
```

Comment:
```suggestion
  "content-manager.components.Filters.usersSelect.label": "Search and select a user to filter by",
```

Reply:
I think this is fine :)

Reply:
I did a quick search that's not very clear